### DISCUSSION TITLE
Help! I'm trying to create a rotation (clock format)

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#5579] [Q&A]
Hello!

Anyone can help me with a step by step configuration to get this playlists loop in AutoDJ:

Music
Promo
Jingle
Music
Spots
Jingle

My current settings for each playlist are:
Music - general rotation playlist
Promo - jingle mode playlist, once per 2 songs
Jingle - jingle mode playlist, once per 1 song
Spots - jingle mode playlist, once per 2 songs. 

But all i get is this order:

Music
Jingle
Promo
Music
Spots
Jingle

Or even this:

Music
Jingle
Promo
Spots
Music
Jingle
Spots
Promo

How do i define the priority for each item. I want the jingle to be always played AFTER promos or spots. 

Maybe someone could help me with a code for the liquidsoft custom configuration. 

Many thanks!

**Comment by Mozitux** [comment]
Hi, you have to create playlists containing your promo files and jingles, then make a sequence as mentioned here: https://github.com/AzuraCast/AzuraCast/discussions/5582

To define the priority of jingle and promo playlist, you must configure the weight of the playlist as being higher than the music one.
